### DISCUSSION TITLE
Two-click report spam/scam button

### DISCUSSION THREAD

**Discussion by SoniEx2** [mastodon/mastodon#28530] [General]
### Pitch

It should be possible to click a button to instantly report spam/scam. The button should be available on posts and profiles and should simply generate a report containing "spam/scam" as the reason. Importantly it should not force the user through any "report system onboarding" bullshit.

### Motivation

The completely useless, artificially generated friction towards reporting spam/scam makes it less likely for folks to report spam/scam. By having a dedicated button for it that skips that whole thing, reporting spam/scam becomes a lot smoother.

Just because the current system is not as bad as github's, doesn't mean it's any good.

**Comment by SoniEx2** [comment]
You used to be able to report spam in 2 clicks. Now you have to be talked down by the user interface before you can report spam. It's condescending.

**Comment by Wild1145** [comment]
As an admin, the majority of reports as it is aren't easy to see why the user reported it, I'm not convinced simplifying the flow for one case (Where Spam and Scam are two very different types of reports with two potentially different outcomes and as an admin I'd want separate in my report view anyway) will actually make things easier, and instead would just lead to more people asking for "Quick click reporting" for other types.

**Comment by SoniEx2** [comment]
Maybe if the report workflow weren't so condescending/trying to push you off from reporting it'd be more bearable.
